FAQ.md

```diff
@@ -84,6 +84,47 @@ TiDB 字符集默认就是 UTF8 而且目前只支持 UTF8，字符串就是 mem
 
 一个事务中的语句数量，默认限制最大为 5000 条。
 
+#### 1.1.19 TiDB 是否支持 XA？
+
+虽然 TiDB 的 JDBC 驱动用的就是 MySQL JDBC（Connector / J）, 但是当使用 Atomikos 的时候，数据源要配置成类似这样的配置：`type="com.mysql.jdbc.jdbc2.optional.MysqlXADataSource"`。MySQL JDBC XADataSource 连接 TiDB 的模式目前是不支持的。MySQL JDBC 中配置好的 XADataSource 模式，只对 MySQL 数据库起作用（dml 去修改 redo 等）。
```

Comment:
- Please pay attention to the punctuation: change the first comma to the Chinese version.
- dml -> DML